webapp/components/post_view/post_info/post_info.jsx

```diff
@@ -392,7 +482,7 @@ export default class PostInfo extends React.Component {
                     {this.createRemovePostButton()}
                 </li>
             );
-        } else if (!isPending) {
+        } else if (!post.failed) {
```

Comment:
We're using the optimistic action in redux so there is no loading state anymore, the client assumes the post succeeded